pkgs/tools/networking/miniupnpd/default.nix

```diff
@@ -15,22 +16,40 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-b9cBn5Nv+IxB58gi9G8QtRvXLWZZePZYZIPedbMMNr8=";
   };
 
-  buildInputs = [ iptables-legacy libuuid openssl ];
+  buildInputs = [ iptables-legacy libuuid openssl ]
+    ++ lib.optionals (firewall == "nftables") [ libmnl libnftnl ];
   nativeBuildInputs= [ pkg-config makeWrapper ];
 
 
   # ./configure is not a standard configure file, errors with:
   # Option not recognized : --prefix=
   dontAddPrefix = true;
 
+  FW = firewall;
+  # allow using various config options
+  IPV6 = true;
```

Comment:
is there any reason those are not makefile options?

Reply:
I remember there being some problem with configureFlags not having any effect, but it works now so I guess I just made a mistake back then